code/lib/cli/src/detect.ts

```diff
@@ -128,14 +128,23 @@ export async function detectBuilder(packageManager: JsPackageManager, projectTyp
 
   // Fallback to Vite or Webpack based on project type
   switch (projectType) {
-    case ProjectType.VUE:
-    case ProjectType.VUE3:
     case ProjectType.SFC_VUE:
       return CoreBuilder.Vite;
     case ProjectType.REACT_SCRIPTS:
     case ProjectType.ANGULAR:
     case ProjectType.NEXTJS:
+      return CoreBuilder.Webpack5;
+    /**
+     * START: Explicit Webpack 5 builder
+     *
+     * These project types are explicitly using Webpack 5 if a vite.config file is not found, because
+     * a vite.config file is necessary to configure these project types to use Vite, which is already
+     * handled above
+     */
     case ProjectType.PREACT:
+    case ProjectType.VUE:
+    case ProjectType.VUE3:
```

Comment:
What will happen for users who are building a component library, where the vite config is handled in a different pacakge?  We handle this by injecting `@vitejs/plugin-vue` into the storybook vite config for them.

Reply:
Why don't we default to Vite in this case?

Reply:
Indeed, it would make sense to remove `Preact`, `Vue` and `Vue3` from the switch-case statement and show a prompt instead.

We just have to solve a problem regarding our sandboxes. The Preact, Vue and Vue3 sandboxes will not build, because a prompt will be shown. We have to adjust the sandbox script to be able to pass cli arguments to the `storybook init` to manually choose a builder and to skip the prompt on CI.